src/System.Buffers/src/Configurations.props

```diff
@@ -6,13 +6,13 @@
       netstandard;
       netcoreapp2.0-Windows_NT;
       netcoreapp2.0-Unix;
-      uap-Windows_NT;
-      uapaot-Windows_NT;
```

Comment:
What's the reason for a UAPAOT version of this?  I don't see any difference in the project.  Is it because we don't guarantee a consistent system.private.corelib between the two?

Reply:
yeah, when we type forward down to corelib we require two configurations. In fact, in order to force this, I believe we still have the wase were the two corelibs have different public keys so that way we make sure that we are typeforwarding to the right one.